health-and-fitness-organizer/src/components/Navbar.module.css

```diff
@@ -1,7 +1,57 @@
+.coName {
+}
+
+.logo {
+  font-size: xxx-large;
+  margin-top: 2px;
+}
+
+.logoContainer {
+  /* font-size: xx-large; */
+}
+
+.logoAndName {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
+
 .navbar {
   display: flex;
-  flex-direction: row;
+  align-items: center;
+  gap: 100px;
+  background-color: black;
+  color: white;
+  height: 50px;
+  padding: 0 1% 0 1%;
+  font-size: x-large;
+  font-weight: bolder;
+  font-style: italic;
```

Comment:
all these font properties can be condensed into the font: shorthand property, something like:
  font: italic x-large bolder auto; 

https://www.w3schools.com/css/css_font_shorthand.asp